plugins/modules/mso_schema_template_external_epg.py

```diff
@@ -260,11 +294,13 @@ def main():
             name=external_epg,
             displayName=display_name,
             vrfRef=vrf_ref,
+            extEpgType=external_epg_type,
```

Comment:
only add to payload if cloud is used. This will prevent breakage in previous MSO versions that don't support it.